[PoC RESULT]: False
[PoC MSG]:
============================== Run PoC ==============================
------------------------------ Standard Output ------------------------------
============================= test session starts ==============================
platform linux -- Python 3.9.23, pytest-8.4.1, pluggy-1.6.0
rootdir: /workspace/snowflake-connector-python
configfile: tox.ini
collected 10 items / 9 deselected / 1 selected

test/unit/test_log_secret_detector.py F                                  [100%]

=================================== FAILURES ===================================
_______________________________ test_mask_token ________________________________

    def test_mask_token():
        long_token = (
            "_Y1ZNETTn5/qfUWj3Jedby7gipDzQs=U"
            "KyJH9DS=nFzzWnfZKGV+C7GopWCGD4Lj"
            "OLLFZKOE26LXHDt3pTi4iI1qwKuSpf/F"
            "mClCMBSissVsU3Ei590FP0lPQQhcSGcD"
            "u69ZL_1X6e9h5z62t/iY7ZkII28n2qU="
            "nrBJUgPRCIbtJQkVJXIuOHjX4G5yUEKj"
            "ZBAx4w6=_lqtt67bIA=o7D=oUSjfywsR"
            "FoloNIkBPXCwFTv+1RVUHgVA2g8A9Lw5"
            "XdJYuI8vhg=f0bKSq7AhQ2Bh"
        )
    
        rsa_key = (
            "-----BEGIN RSA PRIVATE KEY-----\n"
            "MIIBVAIBADANBgkqhkiG9w0BAQEFAASCAT4wggE6AgEAAkEA0pCa0rw1n4GBjylx\n"
            "sBJPVCrsKO7SowkgJ52Lc8K3hMHNKXvYiqwgizbXFBQA27kvpEVSeRQVC3FAPRU5\n"
            "gjtLRwIDAQABAkBHZbz5o9PS6AjUUEs6VpsLgRpersxBeACtLiBw+h9cJfUerR//\n"
            "tTmNsQ9LlamMu2lOlfbO3R2J45ybF7z94A+hAiEA8piucvAlo9YJ4VViQGRTVvr+\n"
            "xZKekSEYRJBn2czeP+kCIQDeMt1PVk/p0NEcNvQMbO0vJ3+U+lITJRwmtJ9Fs1Lj\n"
            "rwIgJeTdkwyaBI6BepY4w7AoKHUKaNgvNqJBxSv9XNMYgEkCIG2rl1YgWOMkAQI3\n"
            "EW/Ml6jtiugiQT5X07Q69F33q5LbAiEArZM7htafpt0RVia+nC9aY+73wpW0Be9e\n"
            "pDz0yVv8s/Q=\n"
            "-----END RSA PRIVATE KEY-----\n"
        )
    
        json_token = (
            "{'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFt"
            "ZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c'}"
        )
    
        masked, masked_str, err_str = SecretDetector.mask_secrets(rsa_key)
>       assert masked
E       assert False

test/unit/test_log_secret_detector.py:114: AssertionError
=========================== short test summary info ============================
FAILED test/unit/test_log_secret_detector.py::test_mask_token - assert False
======================= 1 failed, 9 deselected in 0.08s ========================

------------------------------ Finish Evaluation ------------------------------


[UnitTest RESULT]: None
[UnitTest MSG]:
 None

[Validation TYPE]: validation_fail